deploy/examples/cluster-test.yaml

```diff
@@ -23,6 +23,9 @@ spec:
   mgr:
     count: 1
     allowMultiplePerNode: true
+    modules:
```

Comment:
can you summarize a little bit I always confuse this with dashboard mgr

Reply:
@parth-gr dashobard is a mgr module that calls the orchestrator -- another mgr module-- (rook or cephadm) to get some information such as daemons, devices, etc. Does this answer your question?

Reply:
so to use dashboard we need to always turn it on?

Reply:
@parth-gr not really, some parts of the dashboard don't depend on the orch but others do. So dashboard partially depend on the orchestrator.